src/math/FrustumArray.js

```diff
@@ -0,0 +1,253 @@
+import { WebGLCoordinateSystem } from '../constants.js';
+import { Frustum } from './Frustum.js';
+import { Matrix4 } from './Matrix4.js';
+
+const _projScreenMatrix = /*@__PURE__*/ new Matrix4();
+const _frustum = /*@__PURE__*/ new Frustum();
+
+/**
+ * FrustumArray is used to determine if an object is visible in at least one camera
+ * from an array of cameras. This is particularly useful for multi-view renderers.
+*/
+class FrustumArray {
+
+	/**
+     * Constructs a new frustum array.
```

Comment:
There are some minor formatting issues in this file. I personally always visualize whitespaces in my editor so I can immediately see when spaces and tabs are mixed. Especially since ESLint can't fix this particular formatting issue.